instrumentation/github.com/emicklei/go-restful/config.go

```diff
@@ -21,23 +21,14 @@ import (
 
 // Config is used to configure the go-restful middleware.
 type Config struct {
-	Tracer      oteltrace.Tracer
-	Propagators otelpropagation.Propagators
+	TraceProvider oteltrace.Provider
```

Comment:
If we're setting this (as opposed to where the config field already exists in the other instrumentation changes) can we name it `TracerProvider` instead?

```suggestion
	TracerProvider oteltrace.Provider
```